components/src/molecules/tabs/Tabs.js

```diff
@@ -22,8 +24,41 @@ export default {
     selectTab (selectedTab) {
       this.$emit('click', selectedTab.name)
       this.tabs.forEach(tab => {
-        tab.isActive = (tab.name === selectedTab.name)
+        tab.isActive = selectedTab === this.activeFocusedTab
+          ? tab.name === this.tabs[this.activeFocusedTab].name
+          : tab.name === selectedTab.name
```

Comment:
What do you think about changing this condition to:
```
if (selectedTab === this.activeFocusedTab) {
  tab.isActive = tab.name === this.tabs[this.activeFocusedTab].name
} else {
  tab.isActive = tab.name === selectedTab.name
}
```
for better readibility, it's hard to understand this shorthand ;)